styles/styles.css

```diff
@@ -67,3 +67,41 @@ div[class*='-up'] .con-block.has-bg.text-block.text-list,
 .caas-spacing-m {
   min-height: 500px;
 }
+
+/* Section metadata fix */
+main > .section[class*='-up'] > .content {
+  max-width: initial;
+  margin: 0;
+}
+
+main .section[class*='grid-width-'] {
+  padding-left: unset;
+  padding-right: unset;
+  display: block;
+}
+
+main .section.two-up,
+main .section.three-up,
+main .section.four-up,
+main .section.five-up {
+  display: grid;
+  padding-left: var(--grid-margins-width);
+  padding-right: var(--grid-margins-width);
+}
+
+@media screen and (min-width: 1200px) {
```

Comment:
Should this be combined with the other 1200px media, line 60, or will this be replaced by a Milo solution?

Reply:
There was talk of contributing something to milo after the bacom fix, but I'm not sure if that's the case anymore. I can combine if you think that's best.

Reply:
Up to you, I don't mind it being separated if it serves different purposes.